cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/AbstractNotificationSubscriberService.java

```diff
@@ -118,7 +126,7 @@ protected NotificationSubscriberThread(String topic) {
     @Override
     public void run() {
       // Fetch the last processed message for the topic
-      messageId = Transactionals.execute(transactional, new TxCallable<String>() {
+      messageId = Transactions.executeUnchecked(transactional, new TxCallable<String>() {
```

Comment:
If I executeUnchecked, I don't have to handle the `TransactionFailureException`. What should I do when I handle it?